### PR TITLE
DENG-3161 create full suppression list Accoustic

### PR DESCRIPTION
**Pull request by lelilia**
create a suppression list table consisting of the historic upload and fivetran data 

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title).
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated.
- [ ] When adding a new derived dataset, ensure that data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data can be available in the [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).

For modifications to schemas in restricted namespaces (see [`CODEOWNERS`](https://github.com/mozilla/bigquery-etl/blob/main/CODEOWNERS)):
- [ ] Follow the [change control procedure](https://docs.google.com/document/d/1TTJi4ht7NuzX6BPG_KTr6omaZg70cEpxe9xlpfnHj9k/edit#heading=h.ttegrcfy18ck)

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/DENG-3164)

### CODE REVIEW

**Review by kirill-demtchouk — APPROVED** [review]
The SQL looks good to me! I am not as familiar with the YML in bqetl, someone else should review that piece.

File: sql/moz-fx-data-marketing-prod/acoustic/suppression_list_v1/query.sql [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+SELECT
+  email_id AS external_id,
+  email,
+  "unsubscribed" AS email_subscribe
+FROM
+  suppression_list
+LEFT JOIN
+  contacts
+  USING (email)
+GROUP BY
```
Comment (kik-kik) [inline sql/moz-fx-data-marketing-prod/acoustic/suppression_list_v1/query.sql:35]: Why GROUP BY? Not seeing any aggregation?

File: sql/moz-fx-data-marketing-prod/acoustic/suppression_list_v1/query.sql [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    `moz-fx-data-bq-fivetran.acoustic_sftp.suppression_export_v_1`
+  UNION ALL
+  SELECT
+    email
+  -- historic upload
+  FROM
+    `moz-fx-data-marketing-prod.acoustic.2024-03-20_main_suppression_list`
+),
+suppression_list AS (
+  SELECT DISTINCT
```
Comment (kik-kik) [inline sql/moz-fx-data-marketing-prod/acoustic/suppression_list_v1/query.sql:14]: We could avoid this extra step by just using UNION DISTINCT in the step before?

**Review by kik-kik — APPROVED** [review]
Just some minor comments.

File: sql/moz-fx-data-marketing-prod/acoustic/suppression_list_v1/query.sql (hunk 2 above)
Comment (lelilia) [reply sql/moz-fx-data-marketing-prod/acoustic/suppression_list_v1/query.sql:14]: TIL
updated it

**Pull request merged**